### PR TITLE
New version: gaphor.gaphor version 2.24.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/139417)

### CODE REVIEW

**Comment by stephengillie** [comment]
URL: https://github.com/gaphor/gaphor/releases/download/2.24.0/gaphor-2.24.0-installer.exe 
Status Code: 200

(Automated message - build 751)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> URI: https://github.com/gaphor/gaphor/blob/main/LICENSE.txt, Validation result: Failed, Http status code: NotFound 

(Automated response - build 751.)

**Review by kbdharun — APPROVED** [review]

**Comment by stephengillie** [comment]
> Url validation error
> * manifests\g\gaphor\gaphor\2.24.0
>   * https://github.com/gaphor/gaphor/blob/main/LICENSE.txt
>     * Http status code: NotFound

Verify:
image

File: manifests/g/gaphor/gaphor/2.24.0/gaphor.gaphor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+PackageLocale: en-US
+Publisher: Gaphor Community
+PublisherUrl: https://gaphor.org/
+PublisherSupportUrl: https://gaphor.org/discuss
+Author: gaphor
+PackageName: Gaphor
+PackageUrl: https://github.com/gaphor/gaphor
+License: Apache-2.0 license
+Copyright: Copyright (c) 2024 Gaphor Contributors
+CopyrightUrl: https://github.com/gaphor/gaphor/blob/main/LICENSE.txt
```
Comment (stephengillie) [inline manifests/g/gaphor/gaphor/2.24.0/gaphor.gaphor.locale.en-US.yaml:15]:
```suggestion
# CopyrightUrl: 
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
This field is optional. If there's a better `CopyrightUrl`, feel free to suggest/use it instead. Otherwise this can be omitted.

File: manifests/g/gaphor/gaphor/2.24.0/gaphor.gaphor.locale.en-US.yaml (hunk 1 above)
Comment (kbdharun) [reply manifests/g/gaphor/gaphor/2.24.0/gaphor.gaphor.locale.en-US.yaml:15]: Maybe, link to the new directory instead?

```suggestion
CopyrightUrl: https://github.com/gaphor/gaphor/tree/main/LICENSES

```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**